surround/tests/cli/project_cli/generated_project_test.py

```diff
@@ -37,27 +37,6 @@ def test_remote_from_subdir(self):
         process = subprocess.run(['surround', 'remote'], encoding='utf-8', stdout=subprocess.PIPE, cwd='temp/temp')
         self.assertEqual(process.stdout, "test_remote\n")
 
-    def test_add_from_subdir(self):
```

Comment:
Instead of removing the tests altogether, probably best to update them with the new add....